client-app/shared/catalog/components/image-gallery.vue

```diff
@@ -134,6 +141,26 @@ function setActiveIndex() {
   activeIndex.value = imagesSwiper.value?.realIndex ?? 0;
 }
 
+function navigateToPrevious() {
+  if (imagesSwiper.value && imagesSwiper.value.allowSlidePrev) {
+    imagesSwiper.value.slidePrev();
+  }
+}
+
+function navigateToNext() {
+  if (imagesSwiper.value && imagesSwiper.value.allowSlideNext) {
+    imagesSwiper.value.slideNext();
+  }
+}
+
+function openLightbox() {
+  const currentImage = document.querySelector(".image-gallery__img--active") as HTMLElement;
```

Comment:
Maybe it’s better to use componentId at the root level of the component (or something similar) in case there’s more than one instance of this component on the page.